packages/peregrine/lib/talons/SignIn/useSignIn.js

```diff
@@ -66,7 +66,7 @@ export const useSignIn = props => {
 
                 // Clear all cart/customer data from cache and redux.
                 await apolloClient.clearCacheData(apolloClient, 'cart');
-                await apolloClient.clearCacheData(apolloClient, 'checkout');
+                await apolloClient.clearCacheData(apolloClient, 'customer');
```

Comment:
I'm not sure to understand this change. Is cache data of checkout also in customer?

Reply:
'checkout' does not exist. I must have done the wrong copy paste back then.
It replaces this: https://github.com/magento/pwa-studio/blob/v12.0.0/packages/peregrine/lib/talons/SignIn/useSignIn.js#L71